src/VirtualClient/VirtualClient.Dependencies/AMDGPUDriverInstallation.cs

```diff
@@ -61,6 +63,17 @@ public bool RebootRequired
             }
         }
 
+        /// <summary>
+        /// Vm series on which driver is installed. (e.g. nvv4, v620)
+        /// </summary>
+        public string VmSeries
+        {
+            get
+            {
+                return this.Parameters.GetValue<string>(nameof(AMDGPUDriverInstallation.VmSeries), "nvv4");
```

Comment:
Like we discussed, we need to remove all the Azure-ism out, this should work for all mi25 machine